tools/wake/main.cpp

```diff
@@ -178,9 +182,9 @@ int main(int argc, char **argv) {
     {0, "in", GOPT_ARGUMENT_REQUIRED},
     {'x', "exec", GOPT_ARGUMENT_REQUIRED},
     {0, "job", GOPT_ARGUMENT_REQUIRED},
-    {'i', "input", GOPT_ARGUMENT_FORBIDDEN},
-    {'o', "output", GOPT_ARGUMENT_FORBIDDEN},
-    {0, "label", GOPT_ARGUMENT_FORBIDDEN},
+    {'i', "input", GOPT_ARGUMENT_REQUIRED | GOPT_REPEATABLE_VALUE, input_files.data(), max_pairs},
+    {'o', "output", GOPT_ARGUMENT_REQUIRED | GOPT_REPEATABLE_VALUE, output_files.data(), max_pairs},
+    {0, "label", GOPT_ARGUMENT_REQUIRED},
```

Comment:
It may not be a straightforward implementation (and is probably a separate PR anyway), but do we want to enable repeating `--label` and `--job` to be able to run multiple queries at once?

Reply:
I thought about that but it has some weird semantics. Maybe still work adding though.

Setting different flags should be intersection `--label --failed` -> show me failed jobs that match label while setting the _same_ flag has to be union `--job 1 --job 2` -> "show me jobs 1 and 2"

It gets weird when you mix them `--job 1 --job 2 --job 3 --failed` -> show me only the failed jobs in the set 1,2,3

Reply:
Ah, right, that does get a bit strange.  And multiple `--label`s could actually go both ways -- though, actually, that's the same question we have for `--input` (is it any job which reads *any* of these files, or any job which reads them *all*) so it seems like we're already setting a precedent for union within a single tag and intersection between different ones.

Reply:
Yeah I think we decided that intersection is more useful than union for some of the fields but currently everything is union based...but also for --job and --input/--output union is clearly more useful lol

For instance there was a myth going around that I also fell for that `--failed --last` would intersect the two...but --failed already implied --last and doing both would just repeat the failed jobs...its not even really union.

Reply:
> though, actually, that's the same question we have for --input 

Yeah, right now its union. We can explore changing that later but I think the policy in the short term will be repeated flags are union (even if only input/output can be repeated right now) and different flags are intersection (I swear I created a PR for this but idk where it went)